packages/core/src/editorUtils/editableInPropertyPage.ts

```diff
@@ -1,9 +1,9 @@
-import type { Observable } from "@babylonjs/core";
+import type { Observable } from "@babylonjs/core/Misc/observable";
 
 /**
  * Enum defining the type of properties that can be edited in the property pages in the node editor
  */
-export const enum PropertyTypeForEdition {
+export enum PropertyTypeForEdition {
```

Comment:
The TS option verbatimModuleSyntax = true does not agree with const enum exports?

Reply:
@sebavan If this same change was made to nodeDecorators.ts in babylon core, would there be any consequences?

Reply:
yup you would need to change the way const enum are transpiled with preserveConstEnums in the ts config I guess.